Changelog.md

```diff
@@ -9,6 +9,8 @@
 
 -   The **Enable MCUboot** toggle reflects the MCUboot trait of the connected
     device.
+-   Programming modem firmware with external JLink is now possible on nrf9131
+    and nrf9151
```

Comment:
```suggestion
-   Programming modem firmware with an external J-Link is now possible on nRF9131 DK
    and nRF9151 DK.
```

Reply:
They do not have to be DK a custom kit with that SoC will use an external JLink and will have same issue

```suggestion
-   Programming modem firmware with an external J-Link is now possible on nrf9131
    and nrf9151
```

Reply:
@greg-fer  any commends on the above?

Reply:
All good.